depends/packages/boost.mk

```diff
@@ -1,15 +1,14 @@
 package=boost
-$(package)_version=1_62_0
-$(package)_download_path=http://sourceforge.net/projects/boost/files/boost/1.62.0
+$(package)_version=1_66_0
+$(package)_download_path=http://sourceforge.net/projects/boost/files/boost/1.66.0
```

Comment:
Sourceforge appears to now support https; at least https://sourceforge.net/projects/boost/files/boost/1.66.0/boost_1_66_0.tar.bz2 works (it goes through a bunch of redirects, but they all appear to be https under the sourceforge.net domain). I checked the hash, and also the 1.62.0 hash.

Reply:
Ref #1619, #2888.

Reply:
Since I [don't trust Sourceforge](https://mail.gnome.org/archives/gimp-developer-list/2015-May/msg00144.html), I followed the current Download link from the http://www.boost.org home page (note http :-( ) which now points to https://dl.bintray.com/boostorg/release/1.66.0/source/ . Downloading https://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.bz2 from there gives the same SHA-256 hash.